project.sh

```diff
@@ -435,6 +435,10 @@ is-npm() {
 	file-exists 'package.json'
 }
 
+is-alpine() {
+	file-exists '/etc/os-release' -a grep -q 'Alpine Linux' '/etc/os-release'
```

Comment:
`file-exists` only accepts 1 argument. How does this work...?

Reply:
unless the `-a` is the binary `and` flag for bashes `test` and in which case we should be using `&&`

Reply:
```
[[ file-exists '/etc/os-release' && grep -q 'Alpine Linux' '/etc/os-release' ]]
```

Potentially

```
file-exists '/etc/os-release' && grep -q 'Alpine Linux' '/etc/os-release'
```